deltacat/compute/stats/utils/io.py

```diff
@@ -54,7 +54,7 @@ def read_cached_delta_stats(delta: Delta,
     missed_stats: Optional[DeltaStatsCacheMiss] = DeltaStatsCacheMiss(missed_columns, delta.locator) \
         if missed_columns else None
 
-    return [DeltaStatsCacheResult.of(found_stats, missed_stats), delta]
+    return DeltaStatsCacheResult.of(found_stats, missed_stats)
```

Comment:
Just tested, I think it needs some more changes to make it backwards compatible with pre-existing stats collect() API. Since I'm using annotated delta now which makes accessing fields inside Delta a little bit different. I can work on another revision to enable both Delta and Annotateddelta